core/src/main/resources/schema/infinispan-config-10.0.xsd

```diff
@@ -535,8 +535,11 @@
     <xs:sequence>
       <xs:element name="advanced-externalizer" minOccurs="0" maxOccurs="unbounded">
         <xs:annotation>
-          <xs:documentation>
-            AdvancedExternalizer provides an alternative way to provide externalizers for marshalling/unmarshalling user defined classes that overcome the deficiencies of the more user-friendly externalizer definition  model explained in Externalizer.
+          <xs:documentation>Deprecated since 10.0. Please utilise ProtoStream annotations on Pojos instead by
+            configuring a context-initializer, or specifying a custom marshaller for user types.
+            AdvancedExternalizer provides an alternative way to provide externalizers for marshalling/unmarshalling user
```

Comment:
I don't understand the part after AdvancedExternalizer

Reply:
I agree, that's just what was there before. I think it was copy and pasted from the AdvancedExternalizer interface javadoc. I've updated to try and give a higher level overview.